### PR TITLE
update(tools): add new section for ui/ux

### LINKED ISSUE #305: UI / UX Tools (for tools section)
- Figma
- Zeplin 
- Sketch 
- Material.io

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #305 

### Steps to test
1. Check Ux-UI Tools
1. Useful Links

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by steph222 — DISMISSED** [review]
LGTM!!

File: docs/ux_and_ui.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### What is Sketch?
+
+It is a graphic editor for macOS developed by Dutch Company.
+
+### Why we use Sketch?
+
+It is used for user experience and user interface in web and mobile app design. Some of the features that this technology has is the prototyping and collaboration in the design.
+
+### Sketch Installation?
```
Comment (leisterfrancisco) [inline docs/ux_and_ui.md:60]: Why question mark?

File: docs/ux_and_ui.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+[Sketch Installation](https://www.sketch.com/updates/)
+
+ <iframe
+width="100%" height="315" src="https://www.youtube.com/embed/ilcwjXTqyNM"  frameborder="0" allowfullscreen="true"> </iframe>
+
+* * *
+
+## Material.io
+
+### What is Material.io
```
Comment (leisterfrancisco) [inline docs/ux_and_ui.md:71]: Missing question mark, or well, in others titles, remove that one to make it consistent

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Comment by leisterfrancisco** [comment]
Solve docs/community-resources/useful-links.md conflict

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Review by steph222 — DISMISSED** [review]

File: docs/community-resources/useful-links.md [hunk 3]
```diff
@@ -16,4 +16,11 @@ Here is a list of helpful links related to the EOS community.
 * [EOSIO Training and Certification](https://training.eos.io/)
 * [Kubernetes Documentation](https://kubernetes.io/docs/tutorials/)
 * [Telos Documentation GitHub](https://github.com/telosnetwork/docs)
-* [Learning EOSIO development (Telos, EOS and other blockchains)](https://theblockstalk.medium.com/learning-eosio-development-telos-eos-and-other-blockchains-94d384a8b09f)
\ No newline at end of file
+* [Learning EOSIO development (Telos, EOS and other blockchains)](https://theblockstalk.medium.com/learning-eosio-development-telos-eos-and-other-blockchains-94d384a8b09f) 
+
+**UX/UI**
```
Comment (xavier506) [inline docs/community-resources/useful-links.md:21]: Should we use a header tag?

**Comment by murillojorge** [comment]
Reviewing

File: docs/ux_and_ui.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+## Zeplin
+
+### What is Zeplin?
+
+Zeplin is a plugin and a application for Mac and Windows specialized for helping in the transition between designers and engineers.
+
+### Why we use Zeplin?
+
+Zeplin facilitates the transfer of designs by taking them from Sketch, Adobe XD, Figma, and Photoshop and exporting them into a format that can easily generate code snippets, style guides, specs, and assets.
```
Comment (murillojorge) [inline docs/ux_and_ui.md:39]: It's also used as a source of truth for design deliveries.

File: docs/ux_and_ui.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+<iframe
+width="100%" height="315" src="https://www.youtube.com/embed/x1RPNx8Jsp4" frameborder="0" allowfullscreen="true"> </iframe>
+
+* * *
+
+## Sketch
+
+### What is Sketch?
+
+It is a graphic editor for macOS developed by Dutch Company.
```
Comment (murillojorge) [inline docs/ux_and_ui.md:54]: It is a web design vector-based graphic editor...

**Review by xavier506 — APPROVED** [review]
LGTM 

@Edgar097  did the changes jorge requested get put into the spanish version also?

**Review by steph222 — DISMISSED** [review]

**Pull request merged**